dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -123,8 +125,9 @@ public void open(DataAccessOption... options) throws IOException {
                 }
             } else if (isWriteAccess) {
                 // Creates a new directory as needed for a dataset.
-                if (dataFile.getOwner().getFileSystemDirectory() != null && !Files.exists(dataFile.getOwner().getFileSystemDirectory())) {
-                    Files.createDirectories(dataFile.getOwner().getFileSystemDirectory());
+                if (dataFile.getOwner().getFileSystemDirectory(System.getProperty("dataverse.files.directory")) != null
+                        && !Files.exists(dataFile.getOwner().getFileSystemDirectory(System.getProperty("dataverse.files.directory")))) {
+                    Files.createDirectories(dataFile.getOwner().getFileSystemDirectory(System.getProperty("dataverse.files.directory")));
```

Comment:
I meant to replace all the occurrences of `System.getProperty("dataverse.files.directory")` in FileAccessIO with `filesRootDirectory`. Replacing it in one place doesn't make much sense